string arm_name
geometry_msgs/Pose start_pose
geometry_msgs/Pose goal_pose
string action_type
---

string result
